packages/cli/src/cmds/preview/preview.ts

```diff
@@ -849,4 +875,42 @@ export default class Preview extends YargsCommand {
     }
     return { task: task, startCb: startCb, stopCb: stopCb };
   }
+
+  private prepareTaskNext(
+    taskDefinition: ITaskDefinition,
+    startMessage: string,
+    isWatchTask: boolean,
+    env?: { [key: string]: string }
```

Comment:
When is this `env` used?

Reply:
Actually not used in `prepareTaskNext`, let me remove this parameter for simplification.